core/src/main/java/io/opencensus/tags/TagContext.java

```diff
@@ -31,17 +32,17 @@
  */
 @Immutable
 public final class TagContext {
-  /** The maximum length for a string tag value. */
-  public static final int MAX_STRING_LENGTH = StringUtil.MAX_LENGTH;
+
+  private static final TagContext EMPTY = newBuilder().build();
```

Comment:
Personally I like more the pattern where we make this public instead of having an empty() method. Up to you, in trace package I used this way.